tests/profile.spec.js

```diff
@@ -27,24 +27,11 @@ test("Profile views increase", async ({ page }) => {
   expect(parseInt(startingViews)).toEqual(parseInt(endingViews) - 3);
 });
 
-test("Link clicks increase", async ({ page }) => {
-  await page.goto("/_test-profile-user-4");
-
-  const link = page.locator("text=/Link 1\\s*/i");
-  const startingClicks = (await link.innerText()).match(/(\d+)/g);
-
-  await link.click();
-  await link.click();
-  await link.click();
-
-  await page.waitForResponse(
-    (resp) => resp.url().includes("/api/statistics") && resp.status() === 201
-  );
-
-  const endingClicks = (await link.innerText()).match(/(\d+)/g);
-
-  // 4. get the current link views and see if increased by 3
-  expect(parseInt(startingClicks[1])).toEqual(parseInt(endingClicks[1]) - 3);
+test.fixme("Link clicks increase", async ({ page }) => {
```

Comment:
I am commenting out tests, why? Because this API completely changes that test and it has to be completely rewritten

Reply:
I will make the issue once the PR has been merged 👍

Reply:
We have incomplete tests that need writing but we don't want to remove existing tests otherwise we end up going backwards not forwards - please include existing tests in the PR also

Reply:
Tests added